users/urls.py

```diff
@@ -1,8 +1,9 @@
 from django.urls import path
 
-from .views import LoginView, SignupView
+from .views import LoginView, SignupView, DeleteAccount
 
 urlpatterns = [
     path('/signup', SignupView.as_view()),
     path('/login', LoginView.as_view()),
+    path('/account', DeleteAccount.as_view()),
```

Comment:
```suggestion
    path('', DeleteAccount.as_view()),
```
user 자원을 사용하는 것이므로 위와 같이 수정하면 좋을 것 같습니다.